api/Makefile

```diff
@@ -27,10 +27,11 @@ GO_BUILD_DIR := kmesh.net/kmesh/api
 PROTO_PATH := ../
 PROTO_SRC := $(call find_source, ../api, *.proto)
 
-.PHONY: all install clean
+.PHONY: gen-proto install clean
 
-all:
+gen-proto:
 
+	$(QUIET) bash ../hack/install-proto.sh
 	$(call printlog, PROTO, api/$(C_OUTPUT_DIR))
 	$(QUIET) protoc-c --proto_path=$(PROTO_PATH) --c_out=.  $(PROTO_SRC)
```

Comment:
Do we still need to execute proto-c？

Reply:
yes, this is to gen proto. When we update the proto file, we need to run make gen-proto